packages/okta-vue/src/Auth.js

```diff
@@ -27,7 +27,7 @@ function install (Vue, options) {
       await oktaAuth.signOut()
     },
     async isAuthenticated () {
-      return !!(await oktaAuth.tokenManager.get('accessToken')) || !!(await oktaAuth.tokenManager.get('idToken'))
+      return !!(await this.getAccessToken()) || !!(await this.getIdToken())
```

Comment:
So, I'm wondering how this condition will check if the token is expired

Reply:
Great question @sirgalleto -

Token expiration is handled automatically by our dependency `okta-auth-js` via [automatic token renewal](https://github.com/okta/okta-auth-js/#the-tokenmanager).

If a token is returned by `okta-vue`'s `getAccessToken()` or `getIdToken()` methods, we're stating it hasn't expired and has been [validated](https://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation) (for identity tokens).

Reply:
Exactly.

When `autoRenew` is disabled, we're not going to try and renew the token, and that extra API call to renew it will not happen (and the error will not be thrown). Instead, we [remove the token](https://github.com/okta/okta-auth-js/blob/master/lib/TokenManager.js#L111) from storage, resulting in an `undefined` return value.

Reply:
Nice, thank you for that context. Seems like you have a great team.